curvefs/src/tools/README.md

```diff
@@ -249,6 +249,8 @@ curvefs_tool build-topology
 
 Output:
 
+**if success, there is no output.**  
```

Comment:
Add a TODO in cpp, print some info when exec successfully.

Reply:
> Add a TODO in cpp, print some info when exec successfully.

fix